scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -73,7 +78,7 @@ include::modules/ztp-pgt-config-best-practices.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* For details about best practice for scaling clusters with {rh-rhacm-first}, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/install/installing#performance-and-scalability[ACM performance and scalability considerations].
+* For details about best practice for scaling clusters with {rh-rhacm-first}, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/adoc/install/installing#performance-and-scalability[ACM performance and scalability considerations].
```

Comment:
@aireilly I don't think these links with `adoc` in them will work. Have you tested it in a local build?

Reply:
argh. Good catch. will fix.